CloudForms-Internals/policies/Policy_DROWN Vulnerability.yaml

```diff
@@ -0,0 +1,120 @@
+---
+- MiqPolicy:
+    name: bc031f1a-e249-11e3-9d65-005056b80001
+    description: DROWN Vulnerability
+    expression: !ruby/object:MiqExpression
+      exp:
+        INCLUDES:
+          field: Vm-platform
+          value: linux
+      context_type: 
+    towhat: Vm
+    guid: bc031f1a-e249-11e3-9d65-005056b80001
+    created_by: admin
+    updated_by: admin
+    notes: |
+      This are the openssl package versions affected by DROWN:
+      openssl-1.0.1e-16.el6_5.16
+      openssl-1.0.1e-30.el6_6.12
+      openssl-1.0.1e-51.el7_2.4
+      openssl-1.0.1e-42.el7_1.10
+      openssl-1.0.1e-42.ael7b_1.10
+    active: true
+    mode: compliance
+    read_only: 
+    MiqPolicyContent:
+    - qualifier: failure
+      failure_sequence: 1
+      MiqEventDefinition:
+        name: vm_compliance_check
+        description: VM Compliance Check
+        guid: 42b1bd96-317e-11e3-88e1-005056b80000
+        event_type: Default
+        definition: 
+        default: 
+        enabled: 
+      MiqAction:
+        name: log
+        description: Generate log message
+        guid: e800c9aa-d60f-11e3-85b2-001a4ab4f909
+        action_type: default
+        options: {}
+    - qualifier: failure
+      failure_sequence: 2
+      MiqEventDefinition:
+        name: vm_compliance_check
+        description: VM Compliance Check
+        guid: 42b1bd96-317e-11e3-88e1-005056b80000
+        event_type: Default
+        definition: 
+        default: 
+        enabled: 
+      MiqAction:
+        name: compliance_failed
+        description: Mark as Non-Compliant
+        guid: 339f0cc8-317e-11e3-88e1-005056b80000
+        action_type: default
+        options: {}
+    - qualifier: failure
+      failure_sequence: 3
+      MiqEventDefinition:
+        name: vm_compliance_check
+        description: VM Compliance Check
+        guid: 42b1bd96-317e-11e3-88e1-005056b80000
+        event_type: Default
+        definition: 
+        default: 
+        enabled: 
+      MiqAction:
+        name: d4ea28d8-f051-11e2-97c5-000c297e26aa
+        description: Send Email to Security Team
+        guid: d4ea28d8-f051-11e2-97c5-000c297e26aa
+        action_type: email
+        options:
+          :from: cloudforms_demo@redhat.com
```

Comment:
Do we need to adjust these email addresses to whichever hailstorm domain this appliance is running in?